.gitignore

```diff
@@ -48,3 +48,8 @@ packages/*/LICENSE.md
 packages/enzyme/README.md
 packages/enzyme-adapter-react-*/README.md
 packages/enzyme-adapter-utils*/README.md
+
+# vim
+**/*.swp
+*.swp
```

Comment:
gitignore stuff for a specific editor belongs in your global gitignore, instead of in every repo you touch; please revert this.